### PR TITLE
Has paid column

### PR DESCRIPTION
**Pull request by peteruithoven**
First step of: https://github.com/rikva/voko/issues/63

### CODE REVIEW

File: webapp/accounts/admin.py [hunk 1]
```diff
@@ -142,6 +142,18 @@ def phone(self):
     return self.userprofile.phone_number
 
 
+def has_first_payment(self):
+    try:
+        return Payment.objects.filter(succeeded=True, order__user=self)\
```
Comment (rikva) [inline webapp/accounts/admin.py]: Ik denk dat deze simpeler kan, ongeveer dit:

```
return Payment.objects.filter(succeeded=True, order__user=self).exists()
```

File: webapp/accounts/admin.py [hunk 2]
```diff
@@ -142,6 +142,18 @@ def phone(self):
...
+            .order_by('id').first().created is not None
+    except AttributeError:
+        return False
+
+
+has_first_payment.boolean = True
+has_first_payment.short_description = u"Has payed"
```
Comment (rikva) [inline webapp/accounts/admin.py]: https://www.grammarly.com/blog/paid-payed/

File: webapp/accounts/admin.py (hunk 1 above)
Comment (peteruithoven) [reply webapp/accounts/admin.py]: Oh man, so true. I was struggling with the `list_filter` part that I never looked twice.

File: webapp/accounts/admin.py [hunk 3]
```diff
@@ -147,6 +147,18 @@ def phone(self):
     return self.userprofile.phone_number
 
 
+def has_paid(self):
+    try:
+        return Payment.objects.filter(succeeded=True, order__user=self)\
+            .exists()
+    except AttributeError:
```
Comment (rikva) [inline webapp/accounts/admin.py]: Volgens mij kan die hele try/except nu weg!

**Pull request merged**